src/ZendeskApi.Contracts/Models/Organization.cs

```diff
@@ -49,8 +49,8 @@ public class Organization : IZendeskEntity
         [IgnoreDataMember]
         public long group_id { get; set; }
 
-        [IgnoreDataMember]
-        public long external_id { get; set; }
+        [DataMember(Name = "external_id", EmitDefaultValue = false)]
```

Comment:
Hi! Thank you for your pull request. Could I get you to move this field up to the other non-ignored fields? It helps the readability of the class.